MantidQt/MantidWidgets/src/MantidWSIndexDialog.cpp

```diff
@@ -108,15 +107,15 @@ QMultiMap<QString, std::set<int>> MantidWSIndexWidget::getPlots() const {
  * @returns True if waterfall plot selected
  */
 bool MantidWSIndexWidget::isWaterfallPlotSelected() const {
-  return m_waterfallOpt ? m_waterfallOpt->isChecked() : false;
+  return (m_plotOptions->currentText() == "Waterfall Plot");
```

Comment:
I'm pretty sure `tr("Waterfall Plot") == "Waterfall Plot"` at the moment, since there are no translations to other languages in Mantid. I was just concerned whether this might change... But I guess this won't be an issue in the foreseeable future.